app/models/incident.rb

```diff
@@ -120,6 +120,34 @@ def reviewed?
     staff_reviews.present?
   end
 
+  # from https://gist.github.com/janko-m/2b2cea3e8e21d9232fb9
+  def to_claims_sql
+    report = driver_incident_report
+    fields = {
+      street: report.location,
+      city: report.town,
+      state: 'MA',
+      # zip: driver.incident_report.zip,
+      longitude: longitude,
+      latitude: latitude,
+      'EmployeeID' => driver.badge_number,
+      'IncidentDesc' => root_cause_analysis,
+      'VehicleRouteNum' => report.block,
+      'VehicleNum' => report.bus,
+      'PointOfContact' => report.damage_to_bus_point_of_impact,
```

Comment:
Unrelated to PR, why is this attribute so long when it could be called `PointOfContact`?

Reply:
My feeling is that this attribute name is unambiguous. The impact could have occurred at a different point on the other vehicle, for example.